dashboard/app/models/levels/level.rb

```diff
@@ -844,9 +844,10 @@ def summarize_for_lab2_properties(script, script_level = nil, current_user = nil
     properties_camelized[:appName] = game&.app
     properties_camelized[:useRestrictedSongs] = game.use_restricted_songs?
     properties_camelized[:usesProjects] = try(:is_project_level) || channel_backed?
-    if try(:project_template_level).try(:start_sources) || try(:start_sources)
-      # Override start sources with project template level sources if they exist
-      properties_camelized['startSources'] = try(:project_template_level).try(:start_sources) || try(:start_sources)
+
+    properties_camelized['startSources'] = try(:start_sources)
```

Comment:
do we need this anymore? I think `properties.camelize_keys` will camelize start_sources correctly and put it in properties_camelized.

Reply:
Confirmed that it works fine without it!